src/components/basic-element/types/link.jsx

```diff
@@ -64,7 +65,19 @@ var Link = React.createClass({
     });
   },
 
+  componentDidMount: function () {
+    dispatcher.on('disableLinks', () => {
```

Comment:
I'm a bit worried about these listeners getting added to every link and never getting torn down